aio/aio-proxy/aio_proxy/tests/e2e_tests/test_api.py

```diff
@@ -18,7 +18,7 @@ def test_fetch_company(api_response_tester):
     test if searching for `la poste` returns the right siren as the first search result.
     """
     path = "search?q=la poste"
-    api_response_tester.test_field_value(path, "siren", "356000000")
+    api_response_tester.test_field_value(path, 0, "siren", "356000000")
     api_response_tester.test_number_of_results(path, min_total_results)
```

Comment:
Should not 0 be the default `page_etablissements` value ? Maybe I dont undersatn these `0` you add every where